src/store/actions.ts

```diff
@@ -18,6 +18,22 @@ export * from './market/actions';
 export * from './collectibles/actions';
 
 export const updateStore = () => {
+    return async (dispatch: any, getState: any) => {
+        const state = getState();
+        const currentRoute = getCurrentRoutePath(state);
+        currentRoute.includes(ERC20_APP_BASE_PATH) ? dispatch(updateERC20Store()) : dispatch(updateERC721Store());
+    };
+};
+
+export const updateERC721Store = () => {
+    return async (dispatch: any) => {
+        const web3Wrapper = await getWeb3Wrapper();
+        const [ethAccount] = await web3Wrapper.getAvailableAddressesAsync();
+        dispatch(getAllCollectibles(ethAccount));
+    };
```

Comment:
I believe we need the eth/weth balances too and update the gas info.

Reply:
It's done, please check it again

Reply:
@Agupane I don't see any change related to @unjapones's comment.